app/services/geo_resources/discovery/document_builder/document_path.rb

```diff
@@ -80,9 +80,8 @@ def host
           def map_set_file_set
             member_id = Array.wrap(resource_decorator.thumbnail_id).first
             return unless member_id
-            member = query_service.find_by(id: member_id)
-            member_file_sets = member.decorate.geo_members
```

Comment:
I did not realize that `MapSet` members are `FileSets` retrieved using `#thumbnail_id`

Reply:
This is for building the GeoBlacklight document. GBL docs, especially those that are restricted, need a thumbnail url reference for display in Pulmap. The method above fetches the map set's thumbnail id, which should reference a file set contained in one of the child scanned map documents.